templates/solutions/iot-and-devices/index.html

```diff
@@ -34,7 +34,7 @@
   {%- endif -%}
   {%- if slot == 'image' -%}
     <div class="p-image-container--cinematic is-cover u-hide--small">
-      <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/697a4a90-BRND-2303-IoT%20portfolio%20diagram.svg" alt="" style="object-fit: contain">
+      <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/697a4a90-BRND-2303-IoT%20portfolio%20diagram.svg" alt="A diagram consisting of three wheels in a line, representing the three pillars of Canonical's IoT portfolio: Build. Deploy. Scale. In each wheel, there are four products. These wheels mirror the DevSecOps cycle" style="object-fit: contain">
```

Comment:
Shouldn'it be Build - Secure - Scale? That's what I see in the diagram :thinking: 
```suggestion
      <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/697a4a90-BRND-2303-IoT%20portfolio%20diagram.svg" alt="A diagram consisting of three wheels in a line, representing the three pillars of Canonical's IoT portfolio: Build. Secure. Scale. In each wheel, there are four products. These wheels mirror the DevSecOps cycle" style="object-fit: contain">
```

Reply:
Thanks @lorumic, good catch. updated 🙏